### ISSUE TITLE
Ability to edit alt-text for exhibit content from the SDR in widgets other than item embed

### ISSUE THREAD

**Issue by caaster** [sul-dlss/exhibits#2464]
This is a multi-faceted a11y requirement which **assumes that alt-text for SDR content uploaded to an exhibit is coming from the SDR**. As an exhibit creator, I would expect the following behavior:

- For all SDR content I upload to my exhibit, I assume that the alt-text would be available to screen readers on the Item details page (sul-embed viewer). 
- When I use a widget to display an SDR item I have uploaded to my exhibit, I need to be able to edit the SDR-provided alt-text on each page to accurately reflect the context on each page. The widgets that might be used include: Item row, Item carousel, Item embed, Item slideshow, and possibly Item grid. I may need to make a different alt-text edit to each page where the image is displayed so that the appropriate context can be provided.
- Further investigation needed for alt-text requirements for 3D images, which are currently stored in the SDR but displayed in Sketchfab.

**Comment by caaster** [comment]
I started to edit this ticket for clarity, but I decided to close it because our discussions have evolved considerably -- enough so that this ticket no longer makes sense. It was initially predicated on our assumption that alt text for SDR image content would be stored in the SDR. While that may be true in the future, right now there is no alt text stored for SDR content, nor is there the ability to add alt text for SDR content (and this may or may not be part of the descriptive metadata in the future).
